bedrock/base/templates/includes/protocol/navigation/menus/products.html

```diff
@@ -12,17 +12,12 @@
     <div class="c-menu-panel-container">
       <button class="c-menu-button-close" type="button" aria-controls="c-menu-panel-products">{{ ftl('navigation-v2-close-products-menu') }}</button>
       <div class="c-menu-panel-content">
-        <ul class="mzp-l-rows-four">
+        <ul class="mzp-l-rows-three">
           <li>
             <section class="c-menu-item mzp-has-icon">
-              {% set mozilla_monitor = switch('mozilla-monitor-brand-name') %}
-              <a class="c-menu-item-link" href="https://monitor.mozilla.org/?{{ utm_params }}" data-link-text="{% if mozilla_monitor %}Mozilla Monitor{% else %}Firefox Monitor{% endif %}" data-link-type="nav" data-link-position="topnav" data-link-group="products">
+              <a class="c-menu-item-link" href="https://monitor.mozilla.org/?{{ utm_params }}" data-link-text="Mozilla Monitor" data-link-type="nav" data-link-position="topnav" data-link-group="products">
                 <img loading="lazy" src="{{ static('protocol/img/logos/firefox/monitor/logo.svg') }}" class="c-menu-item-icon" width="32" height="32" alt="">
-                {% if mozilla_monitor %}
-                  <h4 class="c-menu-item-title">{{ ftl('navigation-v2-mozilla-monitor', fallback='navigation-v2-firefox-monitor') }}</h4>
-                {% else %}
-                  <h4 class="c-menu-item-title">{{ ftl('navigation-v2-firefox-monitor', fallback='navigation-firefox-monitor') }}</h4>
-                {% endif %}
+                <h4 class="c-menu-item-title">{{ ftl('navigation-v2-mozilla-monitor', fallback='navigation-v2-firefox-monitor') }}</h4>
```

Comment:
Also you can remove this `fallback` here as well

Reply:
@reemhamz I believe this is the same as above (I have to remove it now with the outdated fluent string pruned), if you meant anything more to be cleaned up than the last commit let me know! Thanks. 🚀

Reply:
Yup you did it right! I just wanted to also mention the `fallback` just in case. Nice job!